sky/cli.py

```diff
@@ -2881,11 +2887,14 @@ def tpunode(cluster: str, yes: bool, port_forward: Optional[List[int]],
 @cli.command()
 @usage_lib.entrypoint
 def check():
-    """Determine the set of clouds available to use.
+    """Check what clouds are available to use.
+
+    This checks access credentials for all clouds supported by SkyPilot. If we
```

Comment:
`If we detect a cloud to be unavailable` -> `If a cloud is detected as unavailable`

Reply:
Done.